Linking the Linux Nightly build fails

https://tools.taskcluster.net/groups/WGhmDYgdQuWlxkOaqFy2Sg/tasks/U1Pdonl0QD2ODOilfaXYxQ/runs/0/logs/public%2Flogs%2Flive.log

```
$ ./mach build --release
[…]
   Compiling libservo v0.0.1 (/repo/components/servo)
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-m64" "-L" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.0.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.1.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.10.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.11.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.12.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.13.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.14.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.15.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.2.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.3.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.4.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.5.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.6.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.7.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.8.rcgu.o" "/repo/target/release/deps/servo-d79c71359c8e8f53.servo.2hvnbg7b-cgu.9.rcgu.o" "-o" "/repo/target/release/deps/servo-d79c71359c8e8f53" "/repo/target/release/deps/servo-d79c71359c8e8f53.33vnujjpauiqn1sn.rcgu.o" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs" "-L" "/repo/target/release/deps" "-L" "/repo/target/release/build/backtrace-sys-8f35c4f0e5a8f0f4/out" "-L" "/repo/target/release/build/libloading-bf39bd49aaf6831c/out" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/repo/target/release/build/azure-fc0c2b0e840f88c2/out/lib" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/repo/target/release/build/servo-skia-6b85fea61b905a9f/out/lib" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/repo/target/release/build/fontsan-88f58e22a87c60ee/out/lib" "-L" "/repo/target/release/build/miniz-sys-076fcefb542d3cdb/out" "-L" "/repo/target/release/build/harfbuzz-sys-2cfbd738e896c483/out/lib" "-L" "/repo/target/release/build/jemalloc-sys-3c1122f969ca2c5e/out/build/lib" "-L" "/repo/target/release/build/mozangle-6187ba28f0d2a0ff/out" "-L" "/repo/target/release/build/mozjs-5103af905c81b0cd/out" "-L" "/repo/target/release/build/mozjs_sys-fdceaea35bb42558/out/js/src/build" "-L" "/repo/target/release/build/mozjs_sys-fdceaea35bb42558/out" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/repo/target/release/build/tinyfiledialogs-6729208300cf2ee8/out" "-L" "/repo/target/release/build/heartbeats-simple-sys-e60c443e1ecf9cea/out/lib" "-L" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/repo/target/release/deps/libservo-d2603b1001d36ae8.rlib" "/repo/target/release/deps/libwebdriver_server-f40b2434a92116c0.rlib" "/repo/target/release/deps/libwebdriver-31a2652f5b5133bb.rlib" "/repo/target/release/deps/libservo_media_gstreamer-5efb2c040cb9b3c3.rlib" "/repo/target/release/deps/libservo_media_gstreamer_render_unix-10c7f3b6ccd45e60.rlib" "/repo/target/release/deps/libgstreamer_gl-a402b6ae5290c5d0.rlib" "/repo/target/release/deps/libgstreamer_gl_sys-0f908839cd6d4af4.rlib" "/repo/target/release/deps/libservo_media_gstreamer_render-9133e8bc99e0ac61.rlib" "/repo/target/release/deps/libgstreamer_webrtc-736f004a2e74738e.rlib" "/repo/target/release/deps/libgstreamer_webrtc_sys-942633a8c6e133ec.rlib" "/repo/target/release/deps/libgstreamer_sdp-a5a91ee6af45a77e.rlib" "/repo/target/release/deps/libgstreamer_sdp_sys-1d7db42d6a4d5bb7.rlib" "/repo/target/release/deps/libgstreamer_player-0188b655e3bec840.rlib" "/repo/target/release/deps/libgstreamer_video-5bf7b4676c3ad4dd.rlib" "/repo/target/release/deps/libgstreamer_player_sys-495545ca10e6a855.rlib" "/repo/target/release/deps/libgstreamer_video_sys-76b6fd35e1a4b643.rlib" "/repo/target/release/deps/libgstreamer_audio-d4a02e6b0a2c5b94.rlib" "/repo/target/release/deps/libgstreamer_audio_sys-774b1cfb9cb5b9c5.rlib" "/repo/target/release/deps/libarray_init-ee552c6b6c0108d3.rlib" "/repo/target/release/deps/libgstreamer_app-20ced87f3220b21a.rlib" "/repo/target/release/deps/libgstreamer_base-2b102d11cb048e03.rlib" "/repo/target/release/deps/libgstreamer_app_sys-4ba675f60766791f.rlib" "/repo/target/release/deps/libgstreamer_base_sys-74b63fd3916862c3.rlib" "/repo/target/release/deps/libgstreamer-092baa647f9ca7bd.rlib" "/repo/target/release/deps/libmuldiv-b8c80c4faf48e233.rlib" "/repo/target/release/deps/libglib-18b0ed85202ae71b.rlib" "/repo/target/release/deps/libgstreamer_sys-6b1457ba043b69f2.rlib" "/repo/target/release/deps/libgobject_sys-258df5b29ed3ec62.rlib" "/repo/target/release/deps/libglib_sys-f9b81729e2011c17.rlib" "/repo/target/release/deps/libenv_logger-e7f1e9067abad785.rlib" "/repo/target/release/deps/libatty-74d5deaa1813d814.rlib" "/repo/target/release/deps/libhumantime-8d440648b0393dcb.rlib" "/repo/target/release/deps/libquick_error-9b4a1ad2a4d2d81a.rlib" "/repo/target/release/deps/libtermcolor-defcf50f1b38334c.rlib" "/repo/target/release/deps/libprofile-3fd8a37670dd8517.rlib" "/repo/target/release/deps/libinfluent-494f157edbe0dcdb.rlib" "/repo/target/release/deps/libheartbeats_simple-4d62eece75ee9277.rlib" "/repo/target/release/deps/libheartbeats_simple_sys-51dce380a76ab3e0.rlib" "/repo/target/release/deps/liblayout_thread-020aad6617c62d68.rlib" "/repo/target/release/deps/libhistogram-2f14e1631a0f2c99.rlib" "/repo/target/release/deps/libscript-e6624593f85016f3.rlib" "/repo/target/release/deps/libenum_iterator-818aacc412638cdd.rlib" "/repo/target/release/deps/libswapper-2e253846350a0df2.rlib" "/repo/target/release/deps/libmozangle-e42bdbeabc349bd0.rlib" "/repo/target/release/deps/libtinyfiledialogs-38e47fa9dc46897a.rlib" "/repo/target/release/deps/libcaseless-9113ecf54a51af24.rlib" "/repo/target/release/deps/libref_filter_map-4324d83b899aaf87.rlib" "/repo/target/release/deps/libref_slice-735aeee8a8bb7e71.rlib" "/repo/target/release/deps/libchrono-fd772182ed4f93c1.rlib" "/repo/target/release/deps/libmitochondria-8419744aab632e00.rlib" "/repo/target/release/deps/libservo_media-0b1b6293ee03b388.rlib" "/repo/target/release/deps/libservo_media_webrtc-4a0d505e8e86284c.rlib" "/repo/target/release/deps/libservo_media_player-72f6c537af99adc7.rlib" "/repo/target/release/deps/libservo_media_streams-31cd7b322fda14e6.rlib" "/repo/target/release/deps/libservo_media_audio-4bcb2c6f346ebbaf.rlib" "/repo/target/release/deps/libpetgraph-ac705105f6daa260.rlib" "/repo/target/release/deps/libordermap-d4234de202e2b45c.rlib" "/repo/target/release/deps/libfixedbitset-3a78009d58816546.rlib" "/repo/target/release/deps/libbyte_slice_cast-c35c4ecab97e8653.rlib" "/repo/target/release/deps/libboxfnonce-c17e4606ef99ae52.rlib" "/repo/target/release/deps/libmozjs-70241825e97432a6.rlib" "/repo/target/release/deps/libmozjs_sys-03288f7c65cee320.rlib" "/repo/target/release/deps/liblibz_sys-8786be3d8ab2c4f0.rlib" "/repo/target/release/deps/liblayout-0a2c5064b182b0f2.rlib" "/repo/target/release/deps/libscript_layout_interface-b155aff0e0f4352d.rlib" "/repo/target/release/deps/libdevtools-3117f1bd2be8b7de.rlib" "/repo/target/release/deps/libconstellation-3fc3cee94294ea0e.rlib" "/repo/target/release/deps/libdebugger-52e5927fec21a05d.rlib" "/repo/target/release/deps/libgaol-df0c61c6b61cb580.rlib" "/repo/target/release/deps/libnet-3695a6d365c1c9b5.rlib" "/repo/target/release/deps/libws-5833dfad125968ea.rlib" "/repo/target/release/deps/libsha1-6578da1cbe4a1d09.rlib" "/repo/target/release/deps/libmio_extras-449dabe9e780e59a.rlib" "/repo/target/release/deps/liblazycell-9acb17a74f1993bb.rlib" "/repo/target/release/deps/libimmeta-cfacd45aa1a86207.rlib" "/repo/target/release/deps/libmime_guess-130af25c42e5bc3d.rlib" "/repo/target/release/deps/liblibflate-38521ccfd2e215fd.rlib" "/repo/target/release/deps/libcrc32fast-cc297137453a6cfa.rlib" "/repo/target/release/deps/libflate2-c955dbf134b3927c.rlib" "/repo/target/release/deps/libminiz_oxide_c_api-b477124647cc9331.rlib" "/repo/target/release/deps/libminiz_oxide-0863289981ac1546.rlib" "/repo/target/release/deps/libcrc-3fa4ab5998b4af24.rlib" "/repo/target/release/deps/libbrotli-980a4eb6638f1d3c.rlib" "/repo/target/release/deps/libbrotli_decompressor-4baac26dacbc135b.rlib" "/repo/target/release/deps/liballoc_stdlib-5a588c81faa0cf4d.rlib" "/repo/target/release/deps/liballoc_no_stdlib-db1c5e55bda3116b.rlib" "/repo/target/release/deps/libbase64-972310a60f9dede2.rlib" "/repo/target/release/deps/libhyper_openssl-bb7af773fc6c4f48.rlib" "/repo/target/release/deps/libtokio_openssl-87a8c607c01bba66.rlib" "/repo/target/release/deps/libopenssl-1299cff0835b7ed0.rlib" "/repo/target/release/deps/libopenssl_sys-69c93af864a8f31e.rlib" "/repo/target/release/deps/libforeign_types-bddb5108ff24522f.rlib" "/repo/target/release/deps/libforeign_types_shared-8805d82ec7bd81aa.rlib" "/repo/target/release/deps/liblinked_hash_set-c7dded4f33e2b638.rlib" "/repo/target/release/deps/liblinked_hash_map-7c306a7849599be7.rlib" "/repo/target/release/deps/libantidote-fd103d97ed92ea15.rlib" "/repo/target/release/deps/libservo_remutex-cb0ad56ded0b3e8d.rlib" "/repo/target/release/deps/liblayout_traits-790398f6a4ea32cd.rlib" "/repo/target/release/deps/libmetrics-375323c6e44dc8a6.rlib" "/repo/target/release/deps/libgfx-9e30f5f46b779b3a.rlib" "/repo/target/release/deps/libfontsan-21091d9e81c2fa6f.rlib" "/repo/target/release/deps/libminiz_sys-b9991f57be358d89.rlib" "/repo/target/release/deps/libucd-93cd8ecb47517336.rlib" "/repo/target/release/deps/libxi_unicode-df0d2db6b2954324.rlib" "/repo/target/release/deps/libfontconfig-f45612d50917d561.rlib" "/repo/target/release/deps/libservo_allocator-b3fd737270a90018.rlib" "/repo/target/release/deps/libjemalloc_sys-e73a491acf87d87b.rlib" "/repo/target/release/deps/libunicode_script-cdc233fe98fbcb67.rlib" "/repo/target/release/deps/libharfbuzz_sys-a75d91810530fa5b.rlib" "/repo/target/release/deps/libstyle-d4544ad8f6ffdbc0.rlib" "/repo/target/release/deps/libuluru-580773ae4af95d88.rlib" "/repo/target/release/deps/libparking_lot-71873644109ac30c.rlib" "/repo/target/release/deps/libparking_lot_core-d31a5560ab3dcfc5.rlib" "/repo/target/release/deps/liblock_api-9a3ef42cb699e76a.rlib" "/repo/target/release/deps/libscopeguard-f38c89505f59caf1.rlib" "/repo/target/release/deps/libitertools-ec606094e57f5a29.rlib" "/repo/target/release/deps/libhtml5ever-4df3e246468b8157.rlib" "/repo/target/release/deps/libfallible-618ea6321dfa5160.rlib" "/repo/target/release/deps/libatomic_refcell-4db8fe2eb06132d9.rlib" "/repo/target/release/deps/libclipboard-97bcc328124e0e83.rlib" "/repo/target/release/deps/libx11_clipboard-82783f58a09628ed.rlib" "/repo/target/release/deps/libxcb-4716fffbbe5fea04.rlib" "/repo/target/release/deps/libcanvas-11c09377a84a34a3.rlib" "/repo/target/release/deps/libhalf-b7398f2e7f36e971.rlib" "/repo/target/release/deps/liboffscreen_gl_context-206a555581af2e28.rlib" "/repo/target/release/deps/libcompositing-b97c9cc36e51d3b2.rlib" "/repo/target/release/deps/libwebvr-d8cf5cedf1358e84.rlib" "/repo/target/release/deps/librust_webvr-efd8a59601d21e9f.rlib" "/repo/target/release/deps/libglutin-d8b0b56d284fd7f1.rlib" "/repo/target/release/deps/libosmesa_sys-5fafb183a52ff7de.rlib" "/repo/target/release/deps/libshared_library-44e2c431d0527757.rlib" "/repo/target/release/deps/libwinit-8ba77e9a8fbf94f4.rlib" "/repo/target/release/deps/libsmithay_client_toolkit-daae4b8f0fd84a31.rlib" "/repo/target/release/deps/libwayland_protocols-162e6917717ab3fe.rlib" "/repo/target/release/deps/libwayland_client-03c8f3de91df6941.rlib" "/repo/target/release/deps/libwayland_commons-fc9f3d597001a36f.rlib" "/repo/target/release/deps/libwayland_sys-2b57f81c7b4dcf31.rlib" "/repo/target/release/deps/libnix-c03ec307471e5e38.rlib" "/repo/target/release/deps/libmemmap-3b815d4488dd9ab2.rlib" "/repo/target/release/deps/libandrew-b0d31c0ad12a277d.rlib" "/repo/target/release/deps/libxml-1bd0d4ffb5d0c780.rlib" "/repo/target/release/deps/libxdg-a84682e059c61515.rlib" "/repo/target/release/deps/libwalkdir-dbc2e668414646c0.rlib" "/repo/target/release/deps/libsame_file-eea8d67dae0bf61b.rlib" "/repo/target/release/deps/librusttype-b35624aa17de17a6.rlib" "/repo/target/release/deps/libstb_truetype-44f5926f46e9e233.rlib" "/repo/target/release/deps/libordered_float-0217c410f1fad3b5.rlib" "/repo/target/release/deps/libapprox-6e952e6d7bc7538e.rlib" "/repo/target/release/deps/libline_drawing-3b0deecd44655cc9.rlib" "/repo/target/release/deps/libdlib-a5b14989ab169207.rlib" "/repo/target/release/deps/liblibloading-03968745309d1eae.rlib" "/repo/target/release/deps/libparking_lot-0f4e2777481d194e.rlib" "/repo/target/release/deps/libparking_lot_core-800b929b0939584f.rlib" "/repo/target/release/deps/liblock_api-98816f716ca41057.rlib" "/repo/target/release/deps/libowning_ref-d5eb68ca618607d0.rlib" "/repo/target/release/deps/libglutin_glx_sys-b420fce2182c37e5.rlib" "/repo/target/release/deps/libx11_dl-083839aa470e1f3f.rlib" "/repo/target/release/deps/libglutin_egl_sys-4e25809b65e44a82.rlib" "/repo/target/release/deps/libscript_traits-6f637bf05d7d3223.rlib" "/repo/target/release/deps/libprofile_traits-9e3b7a0d80048fc1.rlib" "/repo/target/release/deps/libsignpost-5e3614b42b5b9d8f.rlib" "/repo/target/release/deps/libstyle_traits-8ae4bd0976dcad51.rlib" "/repo/target/release/deps/libservo_atoms-f9cbd5a5d75652d3.rlib" "/repo/target/release/deps/libnet_traits-a422e851c8859ce5.rlib" "/repo/target/release/deps/libimage-c8957dd7d94bea1b.rlib" "/repo/target/release/deps/libtiff-5f89d1eb1b502937.rlib" "/repo/target/release/deps/libpng-cbbb14573a84536e.rlib" "/repo/target/release/deps/libdeflate-b8c30eb64f47d288.rlib" "/repo/target/release/deps/libinflate-de24a1fb6881613a.rlib" "/repo/target/release/deps/libadler32-2dfb5144cb2498bb.rlib" "/repo/target/release/deps/libjpeg_decoder-c51dd06d427bad0c.rlib" "/repo/target/release/deps/libscoped_threadpool-648d56f81ddc1e8c.rlib" "/repo/target/release/deps/libgif-573fe2baa81cf3ca.rlib" "/repo/target/release/deps/libcolor_quant-ecf1046343e187ba.rlib" "/repo/target/release/deps/libsafe_transmute-33d2d1404f06d49c.rlib" "/repo/target/release/deps/libnum_rational-444f03a42e719040.rlib" "/repo/target/release/deps/libnum_iter-2749469fba03559b.rlib" "/repo/target/release/deps/libnum_integer-8a32f7f0d30fb0a0.rlib" "/repo/target/release/deps/liblzw-fafa9d97a3608766.rlib" "/repo/target/release/deps/libgfx_traits-503df4836475dbb9.rlib" "/repo/target/release/deps/librange-b90385238e55cb0b.rlib" "/repo/target/release/deps/libdevtools_traits-cb8fed222ad63266.rlib" "/repo/target/release/deps/libwebrender-a16d58e1070d76a2.rlib" "/repo/target/release/deps/libwebrender_build-fde4546e4cf5e1c2.rlib" "/repo/target/release/deps/libsha2-8a507e40b90fa89a.rlib" "/repo/target/release/deps/libdigest-5525292ec298657a.rlib" "/repo/target/release/deps/libopaque_debug-a6b50592a96d886d.rlib" "/repo/target/release/deps/libblock_buffer-7f384cde926fbd5c.rlib" "/repo/target/release/deps/libgeneric_array-18e0dd692479ef12.rlib" "/repo/target/release/deps/libblock_padding-969581fb67269d79.rlib" "/repo/target/release/deps/libbyte_tools-cbd6ad05c84b3aac.rlib" "/repo/target/release/deps/libron-c047fb00b12c81d4.rlib" "/repo/target/release/deps/librayon-1012ae5f76b7ff2a.rlib" "/repo/target/release/deps/libeither-916a765f776afddb.rlib" "/repo/target/release/deps/librayon_core-0082f0514b91c810.rlib" "/repo/target/release/deps/libcrossbeam_deque-391ed6b02b8dbcd8.rlib" "/repo/target/release/deps/libcrossbeam_epoch-4f0639ad2a77ca93.rlib" "/repo/target/release/deps/libcrossbeam_utils-788ae02f43363a89.rlib" "/repo/target/release/deps/libplane_split-71042aa616779956.rlib" "/repo/target/release/deps/libbinary_space_partition-59247bddc49d930b.rlib" "/repo/target/release/deps/libfreetype-bbf9434995b4cd2a.rlib" "/repo/target/release/deps/libthread_profiler-86cf2b4afd6d1af2.rlib" "/repo/target/release/deps/libcstr-263caec75af05b00.rlib" "/repo/target/release/deps/libcanvas_traits-221bd18f56c6b6d9.rlib" "/repo/target/release/deps/libwebvr_traits-9aef85f4f2328f5d.rlib" "/repo/target/release/deps/librust_webvr_api-12dda2b67150fbea.rlib" "/repo/target/release/deps/libpixels-cd0120107383d4ca.rlib" "/repo/target/release/deps/libazure-f8af0e60c164493b.rlib" "/repo/target/release/deps/libskia-4b1ded64c60b4f8e.rlib" "/repo/target/release/deps/libfontconfig_sys-96f383cbc8441e5b.rlib" "/repo/target/release/deps/libexpat_sys-6c328b9927c009bd.rlib" "/repo/target/release/deps/libfreetype_sys-512a5271f28aa92f.rlib" "/repo/target/release/deps/libglx-b8272963a1d4335d.rlib" "/repo/target/release/deps/libx11-134eb159c4744e97.rlib" "/repo/target/release/deps/libgleam-c4e8fae27c4ed3d8.rlib" "/repo/target/release/deps/libbluetooth-27e283af24b819ec.rlib" "/repo/target/release/deps/libservo_config-b2b1dddbf1a2a7b7.rlib" "/repo/target/release/deps/libdirs-9ba48e24b2242207.rlib" "/repo/target/release/deps/libservo_geometry-8c29b760db4510c0.rlib" "/repo/target/release/deps/libgetopts-fb47e71770c025e2.rlib" "/repo/target/release/deps/libbluetooth_traits-e9425f6e70f839ab.rlib" "/repo/target/release/deps/libregex-c3461508bfbd4b18.rlib" "/repo/target/release/deps/libutf8_ranges-09fcedf588b9edaa.rlib" "/repo/target/release/deps/libregex_syntax-9bfb0c203117149d.rlib" "/repo/target/release/deps/libucd_util-0d5826e410791b71.rlib" "/repo/target/release/deps/libthread_local-b9e0cbaa1ede5635.rlib" "/repo/target/release/deps/libaho_corasick-70a80b4ece81f412.rlib" "/repo/target/release/deps/libmemchr-db9ace4f4b7002da.rlib" "/repo/target/release/deps/libembedder_traits-0b68260ed25e0f44.rlib" "/repo/target/release/deps/libservo_url-5239c612929e79cd.rlib" "/repo/target/release/deps/libservo_rand-70778a60cba70324.rlib" "/repo/target/release/deps/liburl_serde-eb7a113946819a3b.rlib" "/repo/target/release/deps/libuuid-84f4820530f335e5.rlib" "/repo/target/release/deps/libdevice-f121105c430a6164.rlib" "/repo/target/release/deps/libblurmock-d5a5a81d690c423f.rlib" "/repo/target/release/deps/libblurz-3dcae681181d769a.rlib" "/repo/target/release/deps/libhex-04eb157e671489be.rlib" "/repo/target/release/deps/libdbus-2d9ecd06fced7c56.rlib" "/repo/target/release/deps/liblibdbus_sys-f70fe43680b6cb07.rlib" "/repo/target/release/deps/libbackground_hang_monitor-ffda5afc93cdd677.rlib" "/repo/target/release/deps/libserde_json-8f32b8355a37b63f.rlib" "/repo/target/release/deps/libunwind_sys-000ba5f784812012.rlib" "/repo/target/release/deps/libnix-38ef23de7c1e729b.rlib" "/repo/target/release/deps/libbacktrace-f2c92bc5bf4338fe.rlib" "/repo/target/release/deps/libbacktrace_sys-0babbe3db9dd1a00.rlib" "/repo/target/release/deps/librustc_demangle-a8c6566ddefac938.rlib" "/repo/target/release/deps/libmsg-6d88fe8bcb8b83f7.rlib" "/repo/target/release/deps/libmalloc_size_of-15ed239370866d97.rlib" "/repo/target/release/deps/libxml5ever-ad6998d05ecdca12.rlib" "/repo/target/release/deps/libmarkup5ever-30386f5e1d59bb81.rlib" "/repo/target/release/deps/libtendril-120fa58767ef19e9.rlib" "/repo/target/release/deps/libutf8-e8f894d7ee25cf17.rlib" "/repo/target/release/deps/libfutf-e11f3fb383641e3b.rlib" "/repo/target/release/deps/libmac-5435619a0af6275f.rlib" "/repo/target/release/deps/libencoding_rs-6b24fb7b56310a7b.rlib" "/repo/target/release/deps/libwebrender_api-9e2586dc8eb2ab86.rlib" "/repo/target/release/deps/libwr_malloc_size_of-3fd761935e60d65f.rlib" "/repo/target/release/deps/liburl-1fc9d617b62aa6c7.rlib" "/repo/target/release/deps/libpercent_encoding-aa0db56d9584ae21.rlib" "/repo/target/release/deps/libidna-ea94d16c89a8a1c4.rlib" "/repo/target/release/deps/libunicode_normalization-364ddb01c9060082.rlib" "/repo/target/release/deps/libunicode_bidi-c0fe8fbafab913ac.rlib" "/repo/target/release/deps/libselectors-5f7dca8f9eef3ded.rlib" "/repo/target/release/deps/libto_shmem-d04a74b97c4decb8.rlib" "/repo/target/release/deps/libstring_cache-32940ca30f83d962.rlib" "/repo/target/release/deps/libstring_cache_shared-a8b5e6a74d81e08b.rlib" "/repo/target/release/deps/libdebug_unreachable-4f535d3a8de3c789.rlib" "/repo/target/release/deps/libsmallbitvec-1cf99d2d8f98a644.rlib" "/repo/target/release/deps/libthin_slice-fb71fc8a6b6cbbdf.rlib" "/repo/target/release/deps/libservo_arc-fd13e3dd4f22d4c2.rlib" "/repo/target/release/deps/libstable_deref_trait-28dbdf4b84fe08cf.rlib" "/repo/target/release/deps/libprecomputed_hash-0e29b563b36972c5.rlib" "/repo/target/release/deps/libfxhash-bdc50d59c8233415.rlib" "/repo/target/release/deps/libkeyboard_types-21826175e7c6230a.rlib" "/repo/target/release/deps/libunicode_segmentation-ac8e3a82896ffd9a.rlib" "/repo/target/release/deps/libhyper_serde-6ba0c98dad415c3e.rlib" "/repo/target/release/deps/libheaders_ext-331570e6b56ad233.rlib" "/repo/target/release/deps/libsha1-d33923ca6e20c1a1.rlib" "/repo/target/release/deps/libfake_simd-3c10f08e34f2376d.rlib" "/repo/target/release/deps/libdigest-296d0fe475596ba5.rlib" "/repo/target/release/deps/libgeneric_array-8641669f9a5e4dcc.rlib" "/repo/target/release/deps/libtypenum-dc9b1962cc2414e9.rlib" "/repo/target/release/deps/libblock_buffer-18e7d235b1a92aae.rlib" "/repo/target/release/deps/libbyte_tools-78f75675e90e0d41.rlib" "/repo/target/release/deps/libarrayref-15bd6fb65d0b2c27.rlib" "/repo/target/release/deps/libheaders_core-6efa2dc3dc7ad26d.rlib" "/repo/target/release/deps/libbitflags-e0d00cd54632d089.rlib" "/repo/target/release/deps/libbase64-3e111f39f4a1b64d.rlib" "/repo/target/release/deps/libsafemem-fa642e0e9b8f93a8.rlib" "/repo/target/release/deps/libserde_bytes-ac8f5a6c9b6c8964.rlib" "/repo/target/release/deps/libmime-a94c05a744a5a5d6.rlib" "/repo/target/release/deps/libunicase-962d6abf7d95720d.rlib" "/repo/target/release/deps/libcookie-64479fb9d8ca99cf.rlib" "/repo/target/release/deps/libhyper-4f5a015ae404d852.rlib" "/repo/target/release/deps/libwant-da25e16c66aa9002.rlib" "/repo/target/release/deps/libtry_lock-effbc9673a227e91.rlib" "/repo/target/release/deps/libtokio-7dc52aa3049487a9.rlib" "/repo/target/release/deps/libtokio_uds-f9d67bc9cce12f00.rlib" "/repo/target/release/deps/libmio_uds-fcfd37b0dac905f2.rlib" "/repo/target/release/deps/libtokio_udp-e3a4e25a56c6e6f4.rlib" "/repo/target/release/deps/libtokio_tcp-fcd03e2a24098986.rlib" "/repo/target/release/deps/libtokio_timer-ace093aac29296ed.rlib" "/repo/target/release/deps/libtokio_reactor-4fc0fbaac85f314c.rlib" "/repo/target/release/deps/libtokio_fs-8dc29639f5f6c8b5.rlib" "/repo/target/release/deps/libtokio_threadpool-ddf5e8de04cbfbe1.rlib" "/repo/target/release/deps/librand-e75494ccdb8c46bf.rlib" "/repo/target/release/deps/libcrossbeam_deque-dbc7666ea19ce470.rlib" "/repo/target/release/deps/libcrossbeam_epoch-5f22f2983510f938.rlib" "/repo/target/release/deps/libscopeguard-db503431494ea67a.rlib" "/repo/target/release/deps/libmemoffset-40f6e94597b0a945.rlib" "/repo/target/release/deps/libcrossbeam_utils-e18ed23b5fa0cb88.rlib" "/repo/target/release/deps/libarrayvec-adaddf293fd61a99.rlib" "/repo/target/release/deps/libnodrop-9bd264925830c696.rlib" "/repo/target/release/deps/libtokio_codec-a2d8c6a441ec70d1.rlib" "/repo/target/release/deps/libtokio_current_thread-3371d504da851eed.rlib" "/repo/target/release/deps/libtokio_executor-041a7d713f684f34.rlib" "/repo/target/release/deps/libtime-f5a74caa5ccab9a3.rlib" "/repo/target/release/deps/libhttparse-519fef1be13c9707.rlib" "/repo/target/release/deps/libh2-8de2045a0fa4eb99.rlib" "/repo/target/release/deps/libindexmap-42054af9b208cfc8.rlib" "/repo/target/release/deps/libstring-ad3986ac489357e1.rlib" "/repo/target/release/deps/libhttp-651a3115167b554c.rlib" "/repo/target/release/deps/libtokio_io-23fd8c9edadca7ed.rlib" "/repo/target/release/deps/libfutures_cpupool-0ddb168d76035075.rlib" "/repo/target/release/deps/libnum_cpus-91ff9fe611067c2d.rlib" "/repo/target/release/deps/libfutures-8e235de5be08ae04.rlib" "/repo/target/release/deps/libbytes-2de3b47d45b57aa2.rlib" "/repo/target/release/deps/libhashglobe-afb8633ded96adc6.rlib" "/repo/target/release/deps/libeuclid-0b4cf21816fc8a50.rlib" "/repo/target/release/deps/libcssparser-f5b93a1343893b22.rlib" "/repo/target/release/deps/libphf-7fadd6753bc8fe6a.rlib" "/repo/target/release/deps/libphf_shared-9a08dff8ca707d2d.rlib" "/repo/target/release/deps/libunicase-b18356fb4c926040.rlib" "/repo/target/release/deps/libsiphasher-42316d4bc2efcba9.rlib" "/repo/target/release/deps/libprocedural_masquerade-29f03f6869962d49.rlib" "/repo/target/release/deps/libmatches-9cfe7bc30fd58100.rlib" "/repo/target/release/deps/libitoa-20a4fab18a54e381.rlib" "/repo/target/release/deps/libdtoa_short-f97cbfbcefc86fcc.rlib" "/repo/target/release/deps/libdtoa-45708138d78d99cc.rlib" "/repo/target/release/deps/libapp_units-9871c965a9e288a6.rlib" "/repo/target/release/deps/libnum_traits-e06c696136bba689.rlib" "/repo/target/release/deps/libipc_channel-15fbaf93d7fc99e5.rlib" "/repo/target/release/deps/libtempfile-97decacf06e9b8c4.rlib" "/repo/target/release/deps/libremove_dir_all-c9ad18439768a607.rlib" "/repo/target/release/deps/librand-a99de4ac1be5c537.rlib" "/repo/target/release/deps/librand-8e48510cc82a0242.rlib" "/repo/target/release/deps/librand_xorshift-7a6af0864616852b.rlib" "/repo/target/release/deps/librand_pcg-7ad7525e7b450d4b.rlib" "/repo/target/release/deps/librand_hc-d2ccd01c8d9c6210.rlib" "/repo/target/release/deps/librand_chacha-d804e90aac2f25f9.rlib" "/repo/target/release/deps/librand_isaac-940ed3f2bcfb7824.rlib" "/repo/target/release/deps/librand_core-fa0a5ac33992b1a7.rlib" "/repo/target/release/deps/librand_os-b70b97689a731287.rlib" "/repo/target/release/deps/librand_jitter-7e090221f824b43f.rlib" "/repo/target/release/deps/librand_core-f6a67978ef7372e9.rlib" "/repo/target/release/deps/libmio-7128fe9a505b87e6.rlib" "/repo/target/release/deps/libslab-5d63646f69d6fe8a.rlib" "/repo/target/release/deps/libiovec-5d98077ab52aa815.rlib" "/repo/target/release/deps/libnet2-9ed64c36c5368a02.rlib" "/repo/target/release/deps/libfnv-a40ff99199b6c491.rlib" "/repo/target/release/deps/libbincode-6ad84d0915fa2d93.rlib" "/repo/target/release/deps/libbyteorder-9d3f3eb4de10ebad.rlib" "/repo/target/release/deps/libcrossbeam_channel-56556a5ef6881f87.rlib" "/repo/target/release/deps/libsmallvec-373a9c96f3b21c94.rlib" "/repo/target/release/deps/libunreachable-cb944f546e001fb2.rlib" "/repo/target/release/deps/libvoid-8a05a574094ab528.rlib" "/repo/target/release/deps/libserde-ba6849fea36045b6.rlib" "/repo/target/release/deps/libcrossbeam_utils-3138e4b1f36a20c8.rlib" "/repo/target/release/deps/libsig-7d6e4fa44999595b.rlib" "/repo/target/release/deps/liblibc-9bdd276e8e67ef86.rlib" "/repo/target/release/deps/liblog-3570f42cb8019428.rlib" "/repo/target/release/deps/libcfg_if-9784b8e63be10e09.rlib" "/repo/target/release/deps/liblazy_static-de3fe6c0100167c9.rlib" "-Wl,--start-group" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-9fb0a673adadad4a.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-b557d18ac04fd677.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libbacktrace_sys-c3211845afa20617.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-9ba6bed47c76febd.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-0854ab83ac6df71c.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-95e3f68476c2e041.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-d9d4e35ceb3fb26b.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-f8be7180a992a02e.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-2a0d064b6bc0a91e.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-261e0420b96a2122.rlib" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-dad47d34f08c7b20.rlib" "-Wl,--end-group" "/root/.rustup/toolchains/nightly-2019-05-14-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-5ed7b22b2c698970.rlib" "-Wl,-Bdynamic" "-lgstgl-1.0" "-lgstvideo-1.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstwebrtc-1.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstsdp-1.0" "-lglib-2.0" "-lgstplayer-1.0" "-lgstvideo-1.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstvideo-1.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstaudio-1.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstapp-1.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstbase-1.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgstreamer-1.0" "-lgobject-2.0" "-lglib-2.0" "-lgobject-2.0" "-lglib-2.0" "-lglib-2.0" "-lstdc++" "-lstdc++" "-lstdc++" "-lz" "-lssl" "-lcrypto" "-lstdc++" "-lpthread" "-lxcb" "-lxcb" "-lxcb" "-lxcb" "-lGL" "-ldl" "-ldl" "-lskia" "-lGL" "-lstdc++" "-lfreetype" "-lbz2" "-lfontconfig" "-lexpat" "-lX11" "-lazure" "-lstdc++" "-lbz2" "-lGL" "-lfontconfig" "-lfreetype" "-lexpat" "-lfreetype" "-lX11" "-ldbus-1" "-llzma" "-lunwind-generic" "-lunwind" "-lutil" "-lutil" "-ldl" "-lrt" "-lpthread" "-lgcc_s" "-lc" "-lm" "-lrt" "-lpthread" "-lutil" "-lutil" "-fuse-ld=gold"
  = note: /repo/target/release/deps/libmozangle-e42bdbeabc349bd0.rlib(RewriteAtomicFunctionExpressions.o):RewriteAtomicFunctionExpressions.cpp:function sh::RewriteAtomicFunctionExpressions(sh::TIntermNode*, sh::TSymbolTable*, int): error: undefined reference to 'void sh::TIntermTraverser::traverse<sh::TIntermNode>(sh::TIntermNode*)'
          collect2: error: ld returned 1 exit status
```

With less scrolling: `error: undefined reference to 'void sh::TIntermTraverser::traverse<sh::TIntermNode>(sh::TIntermNode*)'` in the rlib for the `mozangle` crate.

This wasn’t caught before merging into master by `./mach build --release --with-debug-assertions -p servo` for the WPT tasks: https://build.servo.org/builders/linux-rel-css/builds/12534. Could the environment differences between buildbot and docker-worker have something to do with it?

CC @jdm

--- a/etc/taskcluster/decision_task.py
+++ b/etc/taskcluster/decision_task.py
@@ -42,6 +42,7 @@ def main(task_for):
             linux_arm32_dev,
             linux_arm64_dev,
             linux_wpt,
+            linux_release,
             macos_wpt,
         ]
         by_branch_name = {
@@ -58,7 +59,7 @@ def main(task_for):
             # https://github.com/servo/saltfs/blob/master/homu/map.jinja
 
             "try-mac": [macos_unit],
-            "try-linux": [linux_tidy_unit_docs],
+            "try-linux": [linux_tidy_unit_docs, linux_release],
             "try-windows": [windows_unit, windows_x86, windows_arm64],
             "try-magicleap": [magicleap_dev],
             "try-arm": [linux_arm32_dev, linux_arm64_dev, windows_arm64],
@@ -454,6 +455,17 @@ def linux_nightly():
         .find_or_create("build.linux_x64_nightly" + CONFIG.task_id())
     )
 
+
+def linux_release():
+    return (
+        linux_build_task("Release build")
+        .with_treeherder("Linux x64", "Release")
+        .with_script(
+            "./mach build --release",
+            "./mach package --release",
+        )
+        .find_or_create("build.linux_x64_release" + CONFIG.task_id())
+    )
 
 def linux_wpt():
     release_build_task = (

--- a/etc/taskcluster/decision_task.py
+++ b/etc/taskcluster/decision_task.py
@@ -125,6 +125,8 @@ linux_build_env = {
     "CCACHE": "sccache",
     "RUSTC_WRAPPER": "sccache",
     "SCCACHE_IDLE_TIMEOUT": "1200",
+    "CC": "clang",
+    "CXX": "clang++",
 }
 macos_build_env = {}
 windows_build_env = {

